osu.Game/Rulesets/Judgements/Judgement.cs

```diff
@@ -65,6 +105,21 @@ protected virtual double HealthIncreaseFor(HitResult result)
         {
             switch (result)
             {
+                default:
+                    return 0;
+
+                case HitResult.SmallTickHit:
+                    return DEFAULT_MAX_HEALTH_INCREASE * 0.05;
```

Comment:
We already kinda covered this, but one remaining concern is rulesets that choose to fire way too many ticks may have weird HP distribution. Hopefully the health processor can make sense of this though.

It shouldn't be too bad, just mappers need to be aware of how adding the equivalent of buzz sliders affect hp increase. Which is already a known thing in osu! ruleset.